config/config.py

```diff
@@ -128,28 +153,31 @@ def __init__(self, cli_args, config_json):
 
         self.default_use_cache = config_json.get('default_use_cache', False)
 
-        os_params = config_json.get('os_params')
-        assert config_json.get('os_params') is not None, 'No OS params in config!'
-
-        self.send_to_remote = config_json.get('send_to_remote')
-        self.use_remote_results = config_json.get('use_remote_results', False)
-
-        self.docker_params = {
-            k: v['docker']
-            for k, v in os_params.items()
-            if v.get('docker') is not None and (not self.dist or self.dist == k)
-        }
-        if self.dist_version:
-            assert self.dist_version in self.docker_params[self.dist]['versions'], \
-                f'version {self.dist_version} not found in the list of {self.dist} versions'
-            self.docker_params[self.dist]['versions'] = [self.dist_version]
-
-        self.virtual_box_params = {
-            k: v['virtual_box']
-            for k, v in os_params.items()
-            if v.get('virtual_box') is not None and (not self.dist or self.dist == k)
-        }
-        self.json = config_json
+        if not self.host_mode:
+            os_params = config_json.get('os_params')
+            assert config_json.get('os_params'), 'No OS params in config!'
+
+            self.send_to_remote = config_json.get('send_to_remote', {})
+            self.use_remote_results = config_json.get('use_remote_results', False)
+
+            self.docker_params = {
+                k: v['docker']
+                for k, v in os_params.items()
+                if v.get('docker') is not None and (not self.dist or self.dist == k)
+            }
+            if self.dist_version:
+                assert self.dist_version in self.docker_params[self.dist][
```

Comment:
Please return formatting as it was before.

Reply:
fixed